### PR TITLE
Lms 13243 spring boot

### PR DESCRIPTION
**Pull request by sourin-ghosh**
### **Code Review Feedback changes**

### CODE REVIEW

File: src/main/java/com/example/booksvilla/mapper/UserMapper.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.ReportingPolicy;
+
+@Mapper(
+        componentModel = "spring",
+        unmappedTargetPolicy = ReportingPolicy.ERROR)
+public interface UserMapper {
+
+    @Mapping(source = "id", target = "id")
```
Comment (bojasat01) [inline src/main/java/com/example/booksvilla/mapper/UserMapper.java:16]: if the source, target field name are same , no need to specify this mapping additionally , can you please check for all mapper classes

**Review by bojasat01 — COMMENTED** [review]
please address the review comments

File: src/main/java/com/example/booksvilla/mapper/UserMapper.java (hunk 1 above)
Comment (sourin-ghosh) [reply src/main/java/com/example/booksvilla/mapper/UserMapper.java]: Fixed for all Mappers

**Review by shbojanki — APPROVED** [review]
LGTM

**Pull request merged**